chitchat/src/state.rs

```diff
@@ -1051,12 +1076,17 @@ mod tests {
         let node_state = cluster_state.node_state_mut(&ChitchatId::for_local_test(10_001));
         node_state.heartbeat = Heartbeat(10);
         node_state.set("key", "1");
-        node_state.mark_for_deletion("key");
+        node_state.delete("key");
+        assert!(node_state.get("key").is_none());
         {
             let versioned_value = node_state.get_versioned("key").unwrap();
             assert_eq!(&versioned_value.value, "");
             assert_eq!(versioned_value.version, 2u64);
-            assert!(&versioned_value.is_tombstone());
+            assert!(&versioned_value.is_deleted());
```

Comment:
```suggestion
            assert!(versioned_value.is_deleted());
```
i don't think the borrow is necessary, and i find it somewhat confusing (is it supposed to be a borrow, or a mistyped !)

Reply:
yes!